rn-vector/fonts.gradle

```diff
@@ -6,46 +11,22 @@ afterEvaluate {
     def iconFontsDir = config.iconFontsDir ?: "../../node_modules/@brightlayer-ui/react-native-vector-icons/Fonts";
     def iconFontNames = config.iconFontNames ?: [ "*.ttf" ];
 
-    android.applicationVariants.all { def variant ->
-        def targetName = variant.name.capitalize()
-        def targetPath = variant.dirName
-
-        // Create task for copying fonts
-        def currentFontCopyTask = tasks.create(
-            name: "copy${targetName}ReactNativeBLUIVectorIconFonts",
-            type: Copy) {
-            group = "react"
-            description = "copy fonts into ${targetName}."
-
-            into("$buildDir/intermediates")
-
-            iconFontNames.each { fontName ->
+    def fontCopyTask = tasks.create(
```

Comment:
This is a little overkill since we only have a single font to copy (compared to the 10 or 12 in RNVI), but it doesn't hurt to add some future-proofing, so long as this works fine.